clonelists/Sega - Dreamcast (Redump).json

```diff
@@ -8,7 +8,6 @@
 	"mias": [

```

Comment:
This is included in #80, and I've left comments... so perhaps we can just work on it there.